src/main/java/tds/content/repositories/impl/FileSystemItemDataRepository.java

```diff
@@ -30,18 +31,19 @@
 public class FileSystemItemDataRepository implements ItemDataRepository {
     @Override
     public String findOne(final String itemPath) throws IOException {
-        return Files.toString(new File(itemPath), UTF_8);
+        File itemFile = new File(URLDecoder.decode(itemPath, org.apache.commons.lang3.CharEncoding.UTF_8));
```

Comment:
I guess I'm not following what is going on here.  Why do we need to decode this everywhere in the application?  Can't we decode it in the controller or service?  I guess I'd expect one place to decode it and everywhere else just to act upon it.  Also, maybe the solution is to use a URI rather than passing strings around?

The string was used previously because that is what the legacy code forced.  However, now that we have written a new microservice it seems like String may not be the best representation for this data.